start.py

```diff
@@ -55,7 +56,17 @@ def main(simulation_parameters):
             allagents.do('filobl')
             insurancecustomers.do('check_risk')
 
-        simulation.graphs()
+        if not isTravis:
+            simulation.graphs()
```

Comment:
@davidrpugh It is possible to switch off the GUI. I am doing that in the branches I am developing in [https://github.com/x0range/isle](https://github.com/x0range/isle) using a command line switch taken in with `sys.argv`. But in my opinion it makes sense to have the GUI switched on by default, especially if we plan to at some point deploy this somewhere and make it available to non-programmers.